packages/beacon-node/src/api/impl/beacon/blocks/index.ts

```diff
@@ -224,15 +224,17 @@ export function getBeaconBlockApi({
       () => network.publishBeaconBlock(signedBlock) as Promise<unknown>,
       () =>
         // there is no rush to persist block since we published it to gossip anyway
-        chain.processBlock(blockForImport, {...opts, eagerPersistBlock: false}).catch((e) => {
-          if (e instanceof BlockError && e.type.code === BlockErrorCode.PARENT_UNKNOWN) {
-            network.events.emit(NetworkEvent.unknownBlockParent, {
-              blockInput: blockForImport,
-              peer: IDENTITY_PEER_ID,
-            });
-          }
-          throw e;
-        }),
+        chain
```

Comment:
why did this formatting get updated?

Reply:
There is a subtle difference between Prettier and Biome formatter. specially calling function chaining with anonymous functions. These formatting changes are because of it. 

[See the diff interactively on following link](https://biomejs.dev/playground/?lineWidth=120&indentStyle=space&quoteProperties=preserve&trailingCommas=es5&attributePosition=multiline&bracketSpacing=false&code=ZQB4AHAAbwByAHQAIABmAHUAbgBjAHQAaQBvAG4AIABnAGUAdABCAGUAYQBjAG8AbgBCAGwAbwBjAGsAQQBwAGkAKAB7AAoAIAAgAGMAaABhAGkAbgAsAAoAIAAgAGMAbwBuAGYAaQBnACwACgAgACAAbQBlAHQAcgBpAGMAcwAsAAoAIAAgAG4AZQB0AHcAbwByAGsALAAKACAAIABkAGIALAAKAH0AOgAgAFAAaQBjAGsAPAAKACAAIABBAHAAaQBNAG8AZAB1AGwAZQBzACwACgAgACAAIgBjAGgAYQBpAG4AIgAgAHwAIAAiAGMAbwBuAGYAaQBnACIAIAB8ACAAIgBtAGUAdAByAGkAYwBzACIAIAB8ACAAIgBuAGUAdAB3AG8AcgBrACIAIAB8ACAAIgBkAGIAIgAKAD4AKQA6ACAAQQBwAHAAbABpAGMAYQB0AGkAbwBuAE0AZQB0AGgAbwBkAHMAPAByAG8AdQB0AGUAcwAuAGIAZQBhAGMAbwBuAC4AYgBsAG8AYwBrAC4ARQBuAGQAcABvAGkAbgB0AHMAPgAgAHsACgAgACAAYwBvAG4AcwB0ACAAcAB1AGIAbABpAHMAaABCAGwAbwBjAGsAOgAgAEEAcABwAGwAaQBjAGEAdABpAG8AbgBNAGUAdABoAG8AZABzADwAcgBvAHUAdABlAHMALgBiAGUAYQBjAG8AbgAuAGIAbABvAGMAawAuAEUAbgBkAHAAbwBpAG4AdABzAD4AWwAiAHAAdQBiAGwAaQBzAGgAQgBsAG8AYwBrAFYAMgAiAF0AIAA9ACAAYQBzAHkAbgBjACAAKAAKACAAIAAgACAAewBzAGkAZwBuAGUAZABCAGwAbwBjAGsATwByAEMAbwBuAHQAZQBuAHQAcwAsACAAYgByAG8AYQBkAGMAYQBzAHQAVgBhAGwAaQBkAGEAdABpAG8AbgB9ACwACgAgACAAIAAgAGMAbwBuAHQAZQB4AHQALAAKACAAIAAgACAAbwBwAHQAcwA6ACAAUAB1AGIAbABpAHMAaABCAGwAbwBjAGsATwBwAHQAcwAgAD0AIAB7AH0ACgAgACAAKQAgAD0APgAgAHsACgBjAG8AbgBzAHQAIABwAHUAYgBsAGkAcwBoAFAAcgBvAG0AaQBzAGUAcwAgAD0AIABbAAoAIAAgACAAIAAgACAAIAAgACgAKQAgAD0APgAKACAAIAAgACAAIAAgACAAIAAvAC8AIAB0AGgAZQByAGUAIABpAHMAIABuAG8AIAByAHUAcwBoACAAdABvACAAcABlAHIAcwBpAHMAdAAgAGIAbABvAGMAawAgAHMAaQBuAGMAZQAgAHcAZQAgAHAAdQBiAGwAaQBzAGgAZQBkACAAaQB0ACAAdABvACAAZwBvAHMAcwBpAHAAIABhAG4AeQB3AGEAeQAKACAAIAAgACAAIAAgACAAIABjAGgAYQBpAG4ALgBwAHIAbwBjAGUAcwBzAEIAbABvAGMAawAoAGIAbABvAGMAawBGAG8AcgBJAG0AcABvAHIAdAAsACAAewAuAC4ALgBvAHAAdABzACwAIABlAGEAZwBlAHIAUABlAHIAcwBpAHMAdABCAGwAbwBjAGsAOgAgAGYAYQBsAHMAZQB9ACkACgAgACAAIAAgACAAIAAgACAAIAAgAC4AYwBhAHQAYwBoACgAKABlACkAIAA9AD4AIAB7AAoAIAAgACAAIAAgACAAIAAgACAAIAAgACAAaQBmACAAKABlACAAaQBuAHMAdABhAG4AYwBlAG8AZgAgAEIAbABvAGMAawBFAHIAcgBvAHIAIAAmACYAIABlAC4AdAB5AHAAZQAuAGMAbwBkAGUAIAA9AD0APQAgAEIAbABvAGMAawBFAHIAcgBvAHIAQwBvAGQAZQAuAFAAQQBSAEUATgBUAF8AVQBOAEsATgBPAFcATgApACAAewAKACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIABuAGUAdAB3AG8AcgBrAC4AZQB2AGUAbgB0AHMALgBlAG0AaQB0ACgATgBlAHQAdwBvAHIAawBFAHYAZQBuAHQALgB1AG4AawBuAG8AdwBuAEIAbABvAGMAawBQAGEAcgBlAG4AdAAsACAAewAKACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAYgBsAG8AYwBrAEkAbgBwAHUAdAA6ACAAYgBsAG8AYwBrAEYAbwByAEkAbQBwAG8AcgB0ACwACgAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgAHAAZQBlAHIAOgAgAEkARABFAE4AVABJAFQAWQBfAFAARQBFAFIAXwBJAEQALAAKACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAB9ACkAOwAKACAAIAAgACAAIAAgACAAIAAgACAAIAAgAH0ACgAgACAAIAAgACAAIAAgACAAIAAgACAAIAB0AGgAcgBvAHcAIABlADsACgAgACAAIAAgACAAIAAgACAAIAAgAH0AKQAsAAoAXQA7AAoAIAAgAH0ACgAgACAAfQA7AA%3D%3D)